README.md

```diff
@@ -22,7 +22,8 @@ section, there's a line that says something like:
 libcurl/7.19.5 OpenSSL/1.0.1g zlib/1.2.3.3 libidn/1.15
 ```
 
-Please ensure that your OpenSSL version supports TLS v1.2 or higher.
+Please ensure that your OpenSSL version supports TLS v1.2 (or higher) and that it meets requirements for the [OpenSSL library](http://php.net/manual/en/openssl.requirements.php).
```

Comment:
PHP is now calling these extensions so let's change this a bit to make it clear which requirements we're talking about:
> meets [PHP's requirements for the OpenSSL extension](https://secure.php.net/manual/en/openssl.requirements.php).

Reply:
or "OpenSSL cryptography extension"?

Reply:
Good call, done